02_calculator/calculator_spec.rb

```diff
@@ -77,23 +77,44 @@
 # once the above tests pass,
 # write tests and code for the following:
 
-describe "#multiply" do
+describe "multiply" do
```

Comment:
Rspec convention is actually to keep the `#` to convey that `multiply` is a method.